src/status_im/contexts/wallet/send/transaction_confirmation/view.cljs

```diff
@@ -10,73 +12,168 @@
     [status-im.common.standard-authentication.core :as standard-auth]
     [status-im.contexts.wallet.send.transaction-confirmation.style :as style]
     [utils.i18n :as i18n]
+    [utils.money :as money]
     [utils.re-frame :as rf]
     [utils.security.core :as security]))
 
 (defn- transaction-title
-  [{:keys [token-symbol amount account to-address image-url collectible?]}]
-  [rn/view {:style style/content-container}
-   [rn/view {:style {:flex-direction :row}}
-    [quo/text
-     {:size                :heading-1
-      :weight              :semi-bold
-      :style               style/title-container
-      :accessibility-label :send-label}
-     (i18n/label :t/send)]
-    [quo/summary-tag
-     {:token        (if collectible? "" token-symbol)
-      :label        (str amount " " token-symbol)
-      :type         (if collectible? :collectible :token)
-      :image-source (if collectible? image-url :eth)}]]
-   [rn/view
-    {:style {:flex-direction :row
-             :margin-top     4}}
-    [quo/text
-     {:size                :heading-1
-      :weight              :semi-bold
-      :style               style/title-container
-      :accessibility-label :send-label}
-     (i18n/label :t/from)]
-    [quo/summary-tag
-     {:label               (:name account)
-      :type                :account
-      :emoji               (:emoji account)
-      :customization-color (:color account)}]]
-   [rn/view
-    {:style {:flex-direction :row
-             :margin-top     4}}
-    [quo/text
-     {:size                :heading-1
-      :weight              :semi-bold
-      :style               style/title-container
-      :accessibility-label :send-label}
-     (i18n/label :t/to)]
-    [quo/summary-tag
-     {:type  :address
-      :label (utils/get-shortened-address to-address)}]]])
+  [{:keys [token-symbol amount account to-address route to-network image-url transaction-type
+           collectible?]}]
+  (let [to-network-name  (:network-name to-network)
+        to-network-color (if (= to-network-name :mainnet) :ethereum to-network-name)]
```

Comment:
Insead of having if condition here, maybe where this color is used we can update the color definition from `:ethereum` to `:mainnet` if that is not going to impact other things